pgjdbc/src/test/java/org/postgresql/test/jdbc4/jdbc41/GetObjectTest.java

```diff
@@ -694,6 +694,39 @@ void getBlob() throws SQLException {
     }
   }
 
+  /**
+   * Test the behavior getObject for bytea columns.
+   */
+  @Test
+  void getBytea() throws SQLException {
+    Statement stmt = conn.createStatement();
+    conn.setAutoCommit(false);
```

Comment:
Why `autoCommit` modification is needed here?

Reply:
Well, its still copy-paste from getBlob(), but AFAIK without it the select statement could take too much memory: https://postgrespro.ru/list/thread-id/2502309

Reply:
Right, but you should never run out of memory in this test.